<?xml version="1.0" encoding="utf8"?>
<launch>

  <node name="pixy_cam" pkg="pixy2_ros" type="pixy2_cam" respawn="true" output="screen" >
    <remap from="image_raw" to="/camera/rgb/image"/>
    <remap from="frame_id" to="camera"/>
    <param name="frame_rate" type="int" value="5" />
  </node>

</launch>